### PR TITLE
Added support for extra K8s manifests in the Helm chart

### LINKED ISSUE #6740: [FR] Allow the Chart to create extra manifest
##### What would you like to be added

Sometimes, accessory to the installation there may be manifest to be installed (for example a SecretProviderClass).

It would be nice to be able to do this directly via the Chart using the approach that ArgoCD also uses.

Ref: https://github.com/argoproj/argo-helm/blob/cf57a71a154a1a8a19f5f9d3980d60eb3efd80c8/charts/argo-cd/values.yaml#L62-L87

##### Why is this needed

See above

### PR DESCRIPTION
**Pull request by andreadecorte**
Closes #6740

### CODE REVIEW

File: aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 5.3.1
+version: 5.3.2
```
Comment (desaintmartin) [inline aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml:17]:
```suggestion
version: 5.4.0
```
This is a new feature, according to semver we should bump minor version.

**Review by desaintmartin — CHANGES_REQUESTED** [review]
Thanks for the contribution! Here is a small note.

File: aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml (hunk 1 above)
Comment (andreadecorte) [reply aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml]: makes sense, thanks. Changed

**Review by desaintmartin — APPROVED** [review]

**Comment by desaintmartin** [comment]
/lgtm
/approve 
cc @pierluigilenoci

**Comment by pierluigilenoci** [comment]
@desaintmartin super cool! 🥳 
@andreadecorte thank you a lot! 🚀

**Pull request merged**